modules/services/featureflag/src/main/java/au/com/shiftyjelly/pocketcasts/featureflag/FeatureFlag.kt

```diff
@@ -0,0 +1,29 @@
+package au.com.shiftyjelly.pocketcasts.featureflag
+
+interface Feature {
```

Comment:
I am not going to comment on the renaming but wouldn't it be easier to put this UI under Advanced instead of it's own because that's where I feel like it would avoid the feeling of duplicated items as that area to me dose feel like a beta area regardless.

I'll also point this out because if the wear app needs it's own settings (like auto download top NUM of upnext) I will have to scroll on my big phone again 
(which sure is fine but it is a extra step of thinking after entering a menu in a big phone) 

 ![Screenshot_2023-06-17-12-17-51-69_28ad70af3b47247953fcd94176b9a9c1.jpg](https://github.com/Automattic/pocket-casts-android/assets/20846898/19a5359d-2dfc-4a05-b8b3-ee6723f08e8c)

Reply:
> The only implementation of Feature is FeatureFlag. I don't think we anticipate there being other implementation of it and actually, I think if there were an implementation of Feature that was not a FeatureFlag then things would break because we depend on being able to call FeatureFlag.values()

I was wondering if we needed something like `TestSettings` [here](https://rb.gy/ea4eo) so I kept that structure. But I eliminated the interface and renamed classes in 5283914...4c38b3f till we need anything like it.

Reply:
>  wouldn't it be easier to put this UI under Advanced instead of it's own because that's where I feel like it would avoid the feeling of duplicated items as that area to me dose feel like a beta area regardless.

I'm still thinking about the best place to put this menu. Right now it matches the name and position on the iOS app. But iOS app doesn't have an advanced menu, so I'm still in two minds.

Since it is currently a "debug only" screen right now, let's keep it this way but I'll keep it in mind when it is to be released to the end-users.